sources/packages/backend/libs/integrations/src/services/disbursement-schedule/disbursement-schedule.models.ts

```diff
@@ -215,7 +245,12 @@ export class EligibleECertDisbursement {
     public readonly maxLifetimeBCLoanAmount: number,
     public readonly disabilityDetails: DisabilityDetails,
     readonly restrictions: StudentActiveRestriction[],
-  ) {}
+    readonly restrictionBypass: ApplicationActiveRestrictionBypass[],
```

Comment:
It can be declared as private as it is not intended to be accessed outside right?

Same for 247, but that is not part of the PR.

Reply:
Changed.